# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/gsl/k6_k7.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ode-initval/rk8pd.c.orig  Sat Dec  1 15:10:29 2001
+++ ./ode-initval/rk8pd.c       Sat Dec  1 15:13:07 2001
@@ -30,6 +30,10 @@

 #include "odeiv_util.h"

+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Prince-Dormand constants */

 static const double Abar[] = {
--- ./ode-initval/rkck.c.orig   Tue Jun 12 13:12:36 2001
+++ ./ode-initval/rkck.c        Sat Dec  1 15:14:35 2001
@@ -30,6 +30,10 @@

 #include "odeiv_util.h"

+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Cash-Karp constants */
 static const double ah[] = { 1.0 / 5.0, 0.3, 3.0 / 5.0, 1.0, 7.0 / 8.0 };
 static const double b21 = 1.0 / 5.0;
--- ./ode-initval/rkf45.c.orig  Sat Dec  1 15:39:22 2001
+++ ./ode-initval/rkf45.c       Sat Dec  1 15:40:02 2001
@@ -32,6 +32,10 @@
 
 #include "odeiv_util.h"
 
+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Runge-Kutta-Fehlberg coefficients. Zero elements left out */
 
 static const double ah[] = { 1.0/4.0, 3.0/8.0, 12.0/13.0, 1.0, 1.0/2.0 };
